.github/workflows/performance-benchmarks-bayesmark.yml

```
@@ -59,8 +59,7 @@ jobs:
         python -c 'import optuna'
         optuna --version
 
-        pip install --progress-bar off bayesmark
-        pip install --progress-bar off matplotlib
+        pip install --progress-bar off bayesmark matplotlib pandas
```

I tried to run the benchmark on my fork. I found that we cannot use BoTorchSampler. https://github.com/HideakiImamura/optuna/runs/6682473415?check_suite_focus=true 

How about using `benchmarks` as follow?
```suggestion
        pip install --progress-bar off .[benchmark]
        pip install --progress-bar off bayesmark matplotlib pandas
```

---

Oh, great catch, thanks! I guess the same goes for `cma` (`PyCmaSampler`) and `scikit-optimize` (`SkoptSampler`), but those are defined in `integration` section of requirements. I don't want to pull all integration modules, nor repeat `cma` and `scikit-optimize` in `benchmark` section, so do you think installing them with `matplotlib` and `pandas` is ok here?

---

Fixed in 5ece8c771b87a07ebfe116b25b1051a98fbf48db and tested on [fork](https://github.com/xadrianzetx/optuna-bayesmark/actions/runs/2429401655).